src/js/config.js

```diff
@@ -21,12 +24,13 @@ module.exports = {
   },
 
 
-  STRIPE_API_KEY:                 "",
+  STRIPE_API_KEY: "pk_test_bWuWGC3jrMIFH3wvRvHR6Z5H",
```

Comment:
Hi Steve, since we aren't trying to run build tests on our WeVoteReactNative repository, I wonder if we can move src/js/config.js into .gitignore, and create a src/js/config-template.js that never contains private keys?